src/types.ts

```diff
@@ -35,14 +34,24 @@ type ServiceRequestInit<T = string> = Omit<EnhancedRequestInit<T>, 'method'>
 
 type BaseOptions = {
   headers?: HeadersInit | (() => HeadersInit | Promise<HeadersInit>)
-  requestTransformer?: (request: EnhancedRequestInit) => EnhancedRequestInit | Promise<EnhancedRequestInit>
-  responseTransformer?: (response: TypedResponse) => TypedResponse | Promise<TypedResponse>
+  requestTransformer?: (
+    request: EnhancedRequestInit,
+  ) => EnhancedRequestInit | Promise<EnhancedRequestInit>
+  responseTransformer?: (
+    response: TypedResponse,
+  ) => TypedResponse | Promise<TypedResponse>
```

Comment:
for the sake of extensibility I think it would be nice to define the types of the transformers in `types` and import it here. Also expose them to consumers of the library.

```suggestion
  requestTransformer?: RequestTransformer
  responseTransformer?: ResponseTransformer
```

Reply:
👌🏽 

Done